src/libraries/System.Security.Permissions/src/System/Security/Permissions/Obsoletions.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System
+{
+    internal static class Obsoletions
+    {
+        internal const string SharedUrlFormat = "https://aka.ms/dotnet-warnings/{0}";
```

Comment:
Should be have the Obsoletions.cs file under Common to avoid duplication?

(The IL linker should trim the unused strings literals, so including all strings into each assembly with obsoletions should not be a problem.)